components/org.wso2.carbon.identity.query.saml/src/main/java/org/wso2/carbon/identity/query/saml/handler/SAMLAssertionFinderImpl.java

```diff
@@ -76,42 +79,36 @@ public Assertion findByID(String id) throws IdentitySAML2QueryException {
      * @throws  IdentitySAML2QueryException If unable to collect assertions from database
      */
```

Comment:
please add comments - what this method does...

Reply:
It's already there